trigger_webapp/trigger_app/views.py

```diff
@@ -49,12 +55,60 @@ def home_page(request):
 
 def TriggerEvent_details(request, tid):
     trigger_event = models.TriggerEvent.objects.get(id=tid)
+    # covert ra and dec to HH:MM:SS.SS format
+    c = SkyCoord( trigger_event.ra, trigger_event.dec, frame='icrs', unit=(u.deg,u.deg))
+    trigger_event.ra = c.ra.to_string(unit=u.hour, sep=':')
+    trigger_event.dec = c.dec.to_string(unit=u.degree, sep=':')
+
     voevents = models.VOEvent.objects.filter(trigger_group_id=trigger_event)
-    mwa_obs = models.MWAObservations.objects.filter(trigger_group_id=trigger_event)
+    proj_decs = models.ProjectDecision.objects.filter(trigger_group_id=trigger_event)
+    mwa_obs = []
+    for proj_dec in proj_decs:
+        mwa_obs += models.Observations.objects.filter(project_decision_id=proj_dec)
     return render(request, 'trigger_app/triggerevent_details.html', {'trigger_event':trigger_event,
                                                                      'voevents':voevents,
-                                                                     'mwa_obs':mwa_obs})
+                                                                     'mwa_obs':mwa_obs,
+                                                                     'proj_decs':proj_decs})
+
+def ProjectDecision_details(request, id):
+    proj_dec = models.ProjectDecision.objects.get(id=id)
+
+    # Split message by full stop
+    proj_dec.decision_reason = ".\n".join(proj_dec.decision_reason.split(". "))
```

Comment:
Risky, because ra/dec/time/floats can have a decimal separator. Instead you could not put a '\n' at the end of each message when it's being written, and then strip the final '\n' when complete.

Reply:
I was having a similar thought but more because having to join these each time is annoying. Fixed!